src/CachePlugin.php

```diff
@@ -138,6 +137,40 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
     }
 
     /**
+     * Calculate the timestamp when this cache item should be dropped from the cache. The lowest value return after
+     * the calculation is $maxAge.
+     *
+     * @param int|null $maxAge
+     *
+     * @return int|null Unix system time. @see PSR6 for caching
```

Comment:
Done. 

Sure, private functions should be properly documented. I think one can exclude some edge cases and be a little bit shorter than on public methods. But I guess you are correct. It is a lot of numbers in this class already...